runelite-client/src/main/java/net/runelite/client/plugins/poh/PohConfig.java

```diff
@@ -27,146 +27,186 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.Range;
+import net.runelite.client.config.Units;
 
 @ConfigGroup("poh")
 public interface PohConfig extends Config
 {
```

Comment:
What's the reason for all the reordering happening in config?

Reply:
I added positioning to the entire config menu to allow the burner notifications to be grouped.
It made sense to re-order the config props into numeric position order for easier maintainability moving forwards.

Reply:
I don't see the reason what this has to do with what this pr's point is.

Reply:
They retain the same order in the in-game menu.
They were simply re-ordered in the file to match. Had to add position to all elements for the grouping to work correctly.
As I added in the configurable time limit option after the toggle notification one, it was a bit of a pain to scroll up and down the file finding the next in the numeric sequence.